fuel-vm/src/interpreter/blockchain.rs

```diff
@@ -613,22 +613,22 @@ where
         self.input_contracts.check(&contract_id)?;
 
         // Fetch the storage contract
-        let contract = super::contract::contract(self.storage, &contract_id)?;
-        let contract_bytes = contract.as_ref().as_ref();
-        let contract_len = contract_bytes.len();
         let profiler = ProfileGas {
             pc: self.pc.as_ref(),
             is: self.is,
             current_contract,
             profiler: self.profiler,
         };
+        let contract_len = contract_size(&self.storage, &contract_id)?;
```

Comment:
Does this really save us much expense? We don't know the size of a contract until we load it from disk and check the length of the bytecode slice anyways. 

This is probably going to make us have to load the contract twice from disk in actuality.

Reply:
Yes, we will load the contract twice, but it will be cached on the rocksdb level, so once=) Plus, we don't load the contract into memory to get its size; we use an optimized function that just gets's size from RocksDB.